src/covid_model_seiir_pipeline/pipeline/postprocessing/model/final_outputs.py

```diff
@@ -73,11 +73,63 @@ def __init__(self,
         cumulative_label='cumulative_infections',
         aggregator=aggregators.sum_aggregator,
     ),
+    'r_controlled': MeasureConfig(
+        loaders.load_r_controlled,
+        'r_controlled',
+        splice=False,  # TODO: Get a few rounds of this and then set splice to True
```

Comment:
if all these `splice` args are the same thing then you should set a single variable and reuse it.

Reply:
good call